scripts/test/createProfile.ts

```diff
@@ -13,7 +12,7 @@ export async function createProfile() {
 
     const balance = await ethers.provider.getBalance(deployerAddress);
 
-    const registry = registryConfig[chainId].registryProxy;
+    const registry = "0x4AAcca72145e1dF2aeC137E1f3C5E3D75DB8b5f3";
```

Comment:
^ can we change this back to fetch it from the config ?

Reply:
This doesn't exist on the config currently. We never updated the script when we updated the configs.

Two Options:
1. Just hardcode when we test creating a profile and pool after deploy on testnets.
2. Add the `registryProxy` to the config on all networks (has to be added to the type).